pocs/core.py

```diff
@@ -431,13 +431,14 @@ def sleep(self, delay=2.5, with_status=True):
         if delay > 0.0:
             time.sleep(delay)
 
-    def wait_until_safe(self):
-        """ Waits until weather is safe.
+    def wait_until_dark(self, horizon='astro'):
+        """ Waits until it is dark.
 
```

Comment:
This sounds like a large change in semantics. Which of "is_safe()" or "is_dark()" take weather into account?

Reply:
It is a change in semantics to make it only a wait loop for the different horizon levels. Therefore it is not checking the weather safety at all, which is handled by other methods. The change in name was an attempt to make that more clear.